ExomeDepth/ed_csv_to_vcf.py

```diff
@@ -105,14 +105,12 @@ def cnv_locationtype(region, par1, par2):
             new_record.REF = reference_base
 
             """Write type of call."""
+            new_record.ALT = ["N"]
```

Comment:
ALT "N" is not specified as VCF format specification. https://samtools.github.io/hts-specs/VCFv4.2.pdf 
What does it mean?